src/main/java/com/google/sps/servlets/EditRecipeServlet.java

```diff
@@ -29,13 +36,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String descriptionResponse = request.getParameter("edit-description").trim();
     String ingredientsResponse = request.getParameter("edit-ingredients").trim();
     String stepsResponse = request.getParameter("edit-steps").trim();
-    // privacy will be used once we give users the option to make their recipes public
-    // String privacy = request.getParameter("privacy");
+    String privacy = request.getParameter("privacy");
```

Comment:
Nit: follow your variable naming convention. `privacy` -> `privacyResponse`

Reply:
I only do the response thing above bc everything else has a chance of being changed if they don't enter any values or leave only spaces. privacy is a dropdown so they can't leave something that has to be replaced by the datastore value